subprojects/packagefiles/iir/demo/meson.build

```diff
@@ -0,0 +1,9 @@
+demos = [
+  'iirdemo',
+  'ecg50hzfilt']
+
+foreach demo : demos
+  exe = executable(demo,
```

Comment:
This assignment now does nothing, `exe` is never used.

Before your recent push, it was used to define a `test()`, and the removal of the test is correct since demos aren't necessarily supposed to be tested. (Also, the demos expect to be run in the working directory of the source code, ecg50hzfilt actually segfaults otherwise. Then it goes and writes data files to the source tree, which is *ugly* and definitely should not be triggered by a testsuite. People can manually run the demos if they want. :p)

tl;dr

Clean up from the test removal even more by removing this assignment. :)

Reply:
Done.